domain/src/main/java/com/architectcoders/module/domain/remote_models/Comics/ItemX.kt

```diff
@@ -0,0 +1,6 @@
+package com.architectcoders.module.domain.remote_models.Comics
+
+data class ItemX(
```

Comment:
@joseantoniocampillo we need to avoid the name convention ItemX, ItemXX, and so on because they are difficult to understand